mage/k8s/lib.go

```diff
@@ -266,6 +266,21 @@ func Env() {
 func InstallTool() {
 	loadClusterSettings()
 	clusterActions.InstallTool()
+	installHelm()
```

Comment:
I'm not sure if it makes sense to always download and install helm every time we install the cli tool for kind/k3d/gcloud/etc?

If you move the `installHelm` function below into the `mage/helm/lib.go package ` (maybe also rename the func to `InstallTool`), and then make it a public function, we can add an entry to the root `magefile` to expose it as a top level target so that you can run `mage helm:InstallTool`

I do understand the thought process of "we need a way to ensure all tools at play are installed and using a supported version" but maybe that becomes another mage target entirely? Like a `mage ensureAllTools` or something :thinking:

Reply:
> I'm not sure if it makes sense to always download and install helm every time we install the cli tool for kind/k3d/gcloud/etc?

FWIW, I believe the bash file this downloads doesn't replace helm if the required version already exists. It could also be good to look for the binary in the executable path ahead of time. It sometimes feels to me like we have a lot of mage targets, but if you want more, I can add more.